app/forms/form_base.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class FormBase
```

Comment:
I'd be inclined to rename this to `TraineeFormBase` as it's specifically for Trainees